dj_search/search.py

```diff
@@ -0,0 +1,143 @@
+import datajoint as dj
+import re
+import inspect
+from termcolor import colored
+
+
+class DJSearch:
+
+    def __init__(self, db_prefix='', context=None):
+        self.context = context or inspect.currentframe().f_back.f_locals
+        self.virtual_modules = {}
+
+        self.schema_names = [s for s in dj.list_schemas() if s.startswith(db_prefix)]
```

Comment:
potentially, `db_prefix` could take a list as well?

Reply:
yes, good suggestion

Reply:
or potentially regex (or list of regexes)